client/app/views/partials/pgp/pubkey_display.html

```diff
@@ -1,9 +1,9 @@
 <label data-translate>Details of the PGP key:</label>
 <p>
-  <span data-translate>Fingerprint: </span>
+  <span data-translate>Fingerprint:</span>
```

Comment:
no spaces at the start or end of the internationalized strings; eventually lets keep them outside

is really difficult to that translators whill leave them